app_backend/app/routers/friends.py

```diff
@@ -49,6 +52,13 @@ async def add_friend(resp: Response, friend: AddFriend):
 async def get_friends(resp: Response, token: HTTPAuthorizationCredentials = Security(security)):
     payload = decodeJWT(token)
     if payload is not None:
-        return "test"
+        uuid = payload["user_id"]
+        user = db.get_user_by_uuid(uuid)
+        friends = []
+        for friend_email in user["friends"]:
+            fu = db.get_user_by_email(friend_email)
+            friends.append(json.loads(json_util.dumps(fu)))
```

Comment:
We can optimize this later by having a single batch query. For now, this is good.

Reply:
Do you know the method off the top of your head? I forgot we could do that